config/database.yml

```diff
@@ -15,8 +15,10 @@ development: &default
 
 test:
   <<: *default
-  database: dpul_test
-
+  database: <%= ENV["POMEGRANATE_DB"] %>
+  host: <%= ENV["POMEGRANATE_DB_HOST"] %>
+  username: <%= ENV["POMEGRANATE_DB_USERNAME"] %>
+  password: <%= ENV["POMEGRANATE_DB_PASSWORD"] %>
```

Comment:
Are there any changes needed to run the tests locally?  Could we fallback on the old defaults?

Reply:
My apologies, there were errors in this configuration; I have corrected these, and can now run the tests locally without error.